docs/native-modules-async.md

```diff
@@ -260,16 +260,16 @@ That's it! If you want to see the complete `SimpleHttpModule`, see [`AsyncMethod
 
 ## Executing calls to API on the UI thread
 
-From version 0.64 the native module's code does no longer run on the UI thread. It means that each call to the API that should be executed on the UI thread now needs to be explicitly dispatched.
+Since version 0.64 the native module's code no longer runs on the UI thread. It means that each call to the API that should be executed on the UI thread now needs to be explicitly dispatched.

```

Comment:
```suggestion
Since version 0.64, calls to native modules no longer run on the UI thread. This means that each call to APIs that must be executed on the UI thread now needs to be explicitly dispatched.
```